LandSandBoat/server
# Add nullsong mobskill and made Dahaks able to use it

**_I affirm:_**

## What does this pull request do?

This implements the missing mobskill, Nullsong, which is used by Dahaks. This implements a new function `countEffectWithFlag`. This is needed since Dahak will only use Nullsong if the there are 3 or more effects to dispel. This work is based off of the data collected by Jimmayus in the [Monster Stats Spreadsheet](https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit?gid=57955395#gid=57955395) as well as the [capture done by Melo](https://discord.com/channels/443544205206355968/443894311881408522/1262474594128891957).

## Steps to test these changes

Go fight a dahak and spam `!tp 3000`. It'll never use nullsong. Put some buffs up and then try again and it should eventually use nullsong. Dragons have a lot of mobskills so it can take a while.
